hat/conf/v26.routes

```diff
@@ -70,3 +70,6 @@ GET           /she/function/:function/disable
 GET           /she/function/:function/trigger                                                org.hatdex.hat.she.controllers.FunctionManager.functionTrigger(function: String)
 GET           /she/feed                                                                      org.hatdex.hat.she.controllers.FeedGenerator.fullFeed(since: Option[Long], until: Option[Long])
 GET           /she/feed/$endpoint<[0-9a-z-/]+>                                               org.hatdex.hat.she.controllers.FeedGenerator.getFeed(endpoint: String, since: Option[Long], until: Option[Long])
+
+# Hatters proxy
+POST           /hatters/proxy/*path                                                         org.hatdex.hat.api.controllers.HattersRequestProxy.proxyRequest(path, method: String = "POST")
```

Comment:
Due to the changes requested in the PR #58, please update this endpoint too. Instead of simply proxying the requests, it should consume the claim token (the one with isSignUp value true) AND make a request to hatters claim endpoint. 
Also, it shouldn't be a generic proxy to the hatters service - only make request to the specific claim endpoint.

Reply:
Done